README.md

```diff
@@ -0,0 +1,169 @@
+![Page Glide Logo](images/page_glide_logo.png)
+
+Building an online bookstore requires a strong foundation. 
+Page Glide was created to empower developers with a well-structured and efficient backend API 
+for their online bookstore applications.
+
+## Introduction
+Page Glide is a backend API built with Spring Boot for rapid development. 
+It leverages Spring Security for secure user authentication and authorization. 
+Spring Data JPA provides seamless interaction with a MySQL database for data persistence. 
+Swagger is integrated for clear and comprehensive API documentation. 
+Docker enables easy containerization for deployment.
+
+### Key features
+
+* User Authentication and Authorization (Registration, Login)
+* Book Management (CRUD operations, searching)
+* Category Management (CRUD operations)
+* Shopping Cart Management (adding, updating, removing items)
+* Order Management (placing orders, viewing order history, order details)
+* User Role Management (Admin vs. User functionalities)
+* Unit & Integration API Testing (ensuring code reliability)
+
+## Technologies
+* **Spring Boot**: framework for building Java-based web applications
+* **Spring Security**: security and user management
+* **Spring Data JPA**: data persistence
+* **Swagger**: API documentation
+* **Docker**: deployment and scalability
+* **Liquibase**: database management
+* **MySQL**: database management system
+* **Mockito**: mocking frameworks for tests
+* **JUnit**: testing and reliability
+* **Testcontainers**: in-memory database for integration testing
+
+
+## Installation
+
+### 1. Prerequisites
+
+**Installation without Docker:**
+* Java 21+
+* Maven 4+
+* MySQL 8+
+
+**Installation with Docker:**
+* Docker
+
+### 2. Clone the repository
+```bash
+git clone https://github.com/makaroshyna/online-book-store.git
+cd online-book-store
+```
+
+## Running the project
+
+**Without Docker**
+
+### 1. Set up MySQL
+Establish a new MySQL database, recording its URL, username, and password for future reference.
+
+### 2. Configure environment variables
+Create a file names `application.properties` in `src/main/resources` with the following context
+(replace placeholders with your details):
+```properties
```

Comment:
I sugger to divide instalation in 2 sections like 
-Instalation
    --Docked header
        ---steps
   --No docker header
       ---steps
       
Such structure will be more logical them writting something parentheses